config/class-site-details-index.php

```diff
@@ -191,6 +192,64 @@ private function get_plugin_marketplace( $plugin_url ) {
 		return null;
```

Comment:
I know it was already like that but should this be `null`?

Reply:
Yeah, null is expected for any of the update-related fields that we aren’t able to find.